### PR TITLE
Add react back & integrate with A-Frame

### PR DESCRIPTION
**Pull request by bethqiang**
'Twas oodles of fun.

### CODE REVIEW

File: browser/aframeComponents/scene-load.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* global socket */
+
+import AFRAME from 'aframe';
+
+// This component ensure the scene loads before anything else can happen.
```
Comment (parky22) [inline browser/aframeComponents/scene-load.js]: ensures

File: browser/react/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+// import { Provider } from 'react-redux';
+// import store from '../redux/store';
+import { Router, Route, browserHistory, IndexRedirect } from 'react-router';
+import App from './components/App';
+import 'aframe';
```
Comment (parky22) [inline browser/react/index.js]: let's see if we need this (aframe) and SOCKETS. if not, take out.

File: browser/react/components/App.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import '../../aframeComponents/scene-load';
+
+const App = props => {
```
Comment (JoeyDarbyshire) [inline browser/react/components/App.js]: move the export default up here, possibly?

**Review by JoeyDarbyshire — APPROVED** [review]

File: browser/aframeComponents/scene-load.js (hunk 1 above)
Comment (bethqiang) [inline browser/aframeComponents/scene-load.js]: 👍

File: browser/react/index.js (hunk 2 above)
Comment (bethqiang) [reply browser/react/index.js]: Need sockets. Doesn't seem to need aframe, but I say that with reservation (1st time I tried it without, didn't work. Following two times - without having changed anything else - it was fine). We'll take it out for now and can add it back in later if it's giving us problems.

**Pull request merged**